### PR TITLE
Adding cask for Flume 2.6

### PR DESCRIPTION
**Pull request by qbunt**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

Not entirely sure the delete is working as it should, the paths appear to be correct, but would love your input. Love the project!

### CODE REVIEW

**Comment by qbunt** [comment]
This looks like it's conflicting with a existing formula in homebrew-core. 
https://github.com/Homebrew/homebrew-core/blob/master/Formula/flume.rb
Not sure what to do here, I'll wait for some input from you guys. Thanks!

**Comment by miccal** [comment]
It looks like the Homebrew Formula is unrelated. Audit passes:
```
-bash-4.4.12- /Users/miccal [> brew cask audit --download https://raw.githubusercontent.com/qbunt/homebrew-cask/1108af6e1ba8c3373ea11005cb4afac61d2c9c12/Casks/flume.rb
==> Downloading https://raw.githubusercontent.com/qbunt/homebrew-cask/1108af6e1ba8c3373ea11005cb4afac61d2c9c12/Casks/flume.rb.
######################################################################## 100.0%
==> Downloading https://flumeapp.com/files/Flume-2.6.zip
######################################################################## 100.0%
==> Verifying checksum for Cask flume
audit for flume: passed
```

**Review by miccal — APPROVED** [review]

**Comment by qbunt** [comment]
@miccal 👍  good to hear. Let me know if anything else is needed.

**Comment by vitorgalvao** [comment]
Thank you for the contribution, @qbunt. And welcome.

**Comment by qbunt** [comment]
👍  thanks @vitorgalvao, love this project, I'll definitely contribute again soon.

File: Casks/flume.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'flume' do
+  version '2.6'
+  sha256 '92eb0180efca10d39420bb74856d3c743af9bea3dabc46dd80b25221127838ef'
+
+  url "https://flumeapp.com/files/Flume-#{version}.zip"
```
Comment (rafifyalda) [inline Casks/flume.rb]: 👍

**Pull request merged**